logstash_async/formatter.py

```diff
@@ -299,12 +381,49 @@ def _try_to_get_full_request_uri(self, request):
             return None
 
 
+class DjangoLogstashEcsFormatter(DjangoLogstashFormatter, LogstashEcsFormatter):
+    __schema_dict = {
+        'RESP_STATUS_CODE': 'http.response.status_code',
+        'REQ_USER_AGENT': 'user_agent.original',
+        'REQ_REMOTE_ADDRESS': 'client.ip',
+        'REQ_HOST': 'client.domain',
+        'REQ_URI': 'url.original',
+        'REQ_USER': 'user.name',
+        'REQ_METHOD': 'http.request.method',
+        'REQ_REFERER': 'http.request.referrer',
+    }
+
+    formatter_logstash_message_field_list = (LogstashEcsFormatter.formatter_logstash_message_field_list
+                                             + list(__schema_dict.values()))
+    MessageSchema = type(
+        'MessageSchema',
+        (DjangoLogstashFormatter.MessageSchema, LogstashEcsFormatter.MessageSchema),
+        __schema_dict,
+    )
+
+    def _remove_excluded_fields(self, message, extra_fields):
+        message.pop('status_code', None)
+        super()._remove_excluded_fields(message, extra_fields)
+
+
 class FlaskLogstashFormatter(LogstashFormatter):
+    class MessageSchema(LogstashFormatter.MessageSchema):
+        FLASK_VERSION = 'flask_version'
+        RESP_STATUS_CODE = 'status_code'
+        REQ_USER_AGENT = 'req_useragent'
+        REQ_REMOTE_ADDRESS = 'req_remote_address'
+        REQ_HOST = 'req_host'
+        REQ_URI = 'req_uri'
+        REQ_USER = 'req_user'
+        REQ_METHOD = 'req_method'
+        REQ_REFERER = 'req_referer'
+        REQ_ID = 'request_id'
+        REQ_FORWARDED_PROTO = 'req_forwarded_proto'
+        REQ_FORWARDED_FOR = 'req_forwarded_for'
 
     # ----------------------------------------------------------------------
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self._django_version = None
```

Comment:
Oops, never noticed this typo.

Still, even technically not necessary, I prefer to define instance variables all in `__init__()`, so it would be nice if we change this to 
```
    self._flask_version = None
```

Reply:
Ah, hadn't occurred to me that it was meant to be `self._flask_version`. Done.